src/server/core/Server.js

```diff
@@ -58,10 +60,18 @@ Server.prototype.authorizationHandler = function(request, socket, head)
  */
 Server.prototype.onSocketConnection = function(socket, ip)
 {
-    var client = new SocketClient(socket, 3, ip);
+    // create socket for this client
+    var client = new SocketClient(socket, 3, ip),
+        curvytronController = this.curvytronController;
     this.clients.add(client);
 
     client.on('close', this.onSocketDisconnection);
+
+    // Notify clients with updated nb players
+    client.on('dispatchNbPlayers', function() {
```

Comment:
This is not for the Server to handle that kind of stuff: the controller in charge of "nbPlayers" should listen for client connection/disconnection and send the events itself.